mono/mini/aot-runtime.c

```diff
@@ -4554,7 +4554,7 @@ mono_aot_get_method_checked (MonoDomain *domain, MonoMethod *method, MonoError *
 		}
 
 		if (method_index == 0xffffff) {
-			if (mono_aot_only && mono_trace_is_traced (G_LOG_LEVEL_DEBUG, MONO_TRACE_AOT)) {
+			if (mono_trace_is_traced (G_LOG_LEVEL_DEBUG, MONO_TRACE_AOT)) {
```

Comment:
We should probably only enable this for hybrid aot, its just noice, not finding an instance is not really an error or anything.